quality/findbugs/android.gradle

```diff
@@ -35,6 +35,7 @@ task findbugs(type: FindBugs,
 
 dependencies {
   compileOnly 'com.google.code.findbugs:annotations:3.0.1'
+  compileOnly 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
running `dependencies` i get 
```
compileOnly - Compile only dependencies for 'main' sources. (n)
+--- com.google.code.findbugs:annotations:3.0.1 (n)
\--- com.google.code.findbugs:jsr305:3.0.2 (n)
[..]
debugCompileClasspath - Resolved configuration for compilation for variant: debug
+--- com.google.code.findbugs:annotations:3.0.1
|    +--- net.jcip:jcip-annotations:1.0
|    \--- com.google.code.findbugs:jsr305:3.0.1 -> 3.0.2
+--- com.google.code.findbugs:jsr305:3.0.2
```
so that jar should be present for compilation already (in version 3.0.1) - is the fix in the patch version 3.0.1->3.0.2?